recipes/mariadb-connector-c/meta.yaml

```diff
@@ -0,0 +1,52 @@
+{% set name = "mariadb-connector-c" %}
+{% set version = "3.2.4" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/mariadb-corporation/mariadb-connector-c/archive/v{{ version }}.tar.gz
+  sha256: 6da2563a7940457846267989732b97028c350d7de07f89e355f15646641d0ed7
+
+build:
+  number: 0
+  missing_dso_whitelist:
+    - '*'
```

Comment:
Same here

Reply:
fails on osx when I remove dso_whitelist due to a OverLinkingError